components/create/CreateModal.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <NeoModal
+    append-to-body
+    :value="isModalActive"
+    @close="isModalActive = false"
+  >
+    <NeoModalHead
+      :title="$t('create')"
+      @close="isModalActive = false"
+    />
+    <div class=" px-6 pt-6 pb-5">
+      <div class="space-y-4">
+        <div
+          v-for="item in createOptionsConfig"
+          :key="item.path"
+          :data-testid="item.testId"
+          class="cursor-pointer max-w-[316px] w-full bg-k-grey-light border border-k-grey-light hover:bg-blue-light-cards hover:border-k-blue-hover px-[20px] py-2 flex items-center group"
+          @click="onClickCreateOption(item.path)"
+        >
+          <KIcon
+            :name="item.icon"
+            size="medium"
+            class="text-k-grey group-hover:text-k-blue"
+          />
+          <div class="text-left ml-4">
+            <div class="">
+              {{ item.title }}
+            </div>
+            <div class="text-sm text-k-grey">
+              {{ item.description }}
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="text-center text-sm text-k-grey mt-[20px]">
+        {{ $t('mint.modal.availableOn', ['Ahp', 'Ahk']) }}
```

Comment:
iirc we have a constant with prefix with create enabled, let's use that instead

bug: 

base should not be there 

<img src=https://github.com/user-attachments/assets/740c9a87-6774-4768-9550-08213f383b44 width=400 />

Reply:
updated. I could not find the old constant, by the way, but I created a new one.